src/Gold.Redis/Gold.Redis.LowLevelClient/Communication/RedisSocketAuthenticator.cs

```diff
@@ -15,8 +16,10 @@ public RedisSocketAuthenticator(ISocketCommandExecutor socketCommandExecutor)
         public async Task<bool> TryAuthenticate(Socket connectionSocket, string password)
         {
             var authCommand = "AUTH " + password;
-            var response = await _socketCommandExecutor.ExecuteCommand(connectionSocket, authCommand);
-            return response == Constants.OkResponse;
+            var response = await _socketCommandExecutor.ExecuteCommand<Response>(connectionSocket, authCommand);
+            if(response is SimpleStringResponse simpleResponse)

```

Comment:
There are alot of commands that just return "Ok" -> 
shouldn't we add an BoolResponse ?

Reply:
Ok command, and all the other saved key words in redis fall under the SimpleStringResponse.
I didnt reinvent anything here, only putted the redis documentation in classes.
Now that we have classes for every response type, we can make some static props for default return values but it seems a bit like an over kill for now.